packages/@okta/vuepress-site/docs/guides/common-hook-set-up-steps/adding-basic-auth/index.md

```diff
@@ -13,9 +13,11 @@ To add the `npm` packages:
 2. From the `Add Package` drop-down, search for the `express-basic-auth` and `body-parser` packages.
```

Comment:
@brianduffield-okta I never had to do this for any of the Glitch examples for the Password, Registration and Token Inline hook that I stepped through.  Is this just to tell customers how to setup Basic Auth for a bare-bones Glitch project?

Reply:
Yes. If you copy the Glitch project (re-mix it), there's no need to add these packages and code; it's already there. The topic also provided a little context - I hoped - about how this worked.

I revised as per Ian's comment because I introduced some granular content that had not really been explained at this point, and provided a bit more of a high-level overview of why we were introducing Basic Auth, and then addressed the format of the secret, which was a stumbling point for myself, initially, and a couple of other reviewers (i.e., forgetting to add the scheme "Basic" to the secret).

Reply:
That said, re-reading my topic from the top, I can see there might be some confusion with my opening line. I'll revise.